run_tests

```diff
@@ -4,19 +4,24 @@
 
 USAGE="usage: $0
        -h    show help
-       -f    fix linting errors"
+       -f    fix linting errors
+       -u    run using Ubunut host native packages"
```

Comment:
Ubunut -> Ubuntu

Reply:
Ops. @dstathis may I request you to just "suggest an edit" because if I make another push to fix this, you may need to re-review and re-approve. I think if you "suggest an edit" and I accept this problem does not occur.